migrations/README.adoc

```diff
@@ -7,6 +7,35 @@ The latest version of Fusion 5 is 5.3.0. The migration process varies depending
 
 // tag::body[]
 
+== Migration from Fusion 5.3.x to 5.4.0
+
+==== Prerequisites
+
+Make sure you're fusion admin is running and all collections are healthy
+
+You *should not* be actively making changes to the `signals` collections (via Rules UI) during the upgrade process.
+
+For production systems, this upgrade process should be performed during a maintenance window.
+
+===== Update signal collections solr config
+
+Lucidworks recommends taking a backup of your signal collections just in case something goes wrong with the upgrade, especially for production environments.
+
+Depending on your Ingress config, the export may take too long and timeout. Consequently, we recommend opening a `kubectl` port-forward to the Fusion Gateway pod:
+```
+kubectl port-forward <POD> 6764
+```
+
+Run the below commands in the terminal or update the creds in the update_signal_collection.sh and run ./update_signal_collection.sh
+```
+PROXY="http://localhost:6764"
+APP="YOUR_FUSION_APP_ID"
+curl -u $CREDS -X POST "$PROXY/api/update/all/signalCollection" > ${APP}_signal_collection_update.json
```

Comment:
does `api/update/all/signalCollection` return any json response? if so do we need to save it, since it doesn't look like we are using it later

Reply:
No it doesn't return anything. Logs are printed in fusion-admin

Reply:
ok, I guess we should remove `> ${APP}_signal_collection_update.json` part then, since it won't contain anything useful

Reply:
Yes makes sense. Removed it.